src/common/transformations/tests/op_conversions/convert_topk11_downgrade_test.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <ngraph/pass/manager.hpp>
+#include <openvino/opsets/opset11.hpp>
+#include <openvino/opsets/opset3.hpp>
+#include <transformations/op_conversions/convert_topk11_downgrade.hpp>
+#include <transformations/utils/utils.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace testing;
+
+TEST_F(TransformationTestsF, ConvertTopk11ToTopk3) {
```

Comment:
Please add additional test where TopK11 is not converted (stable_sort is true).
In such test the only the first part is needed, the expected value (reference model) is automatically deduced to itself.